### PR TITLE
feat: replace invoices by bolt11

### LINKED ISSUE #809: Replace npm package to parse lightning invoices
**Is your feature request related to a problem? Please describe.**
Seems the [invoices package](https://www.npmjs.com/package/invoices) keeps having breaking changes in patch releases: 
https://github.com/getAlby/lightning-browser-extension/pull/808
It also comes with heavy dependencies. 

We should replace the package with a different more lightweight bolt11 parser. 

https://www.npmjs.com/search?q=bolt11
https://www.npmjs.com/package/light-bolt11-decoder
https://www.npmjs.com/package/bolt11

### PR DESCRIPTION
**Pull request by dylancom**
#### Link this PR to an issue
Fixes #809

#### Type of change (Remove other not matching type)

- New feature (non-breaking change which adds functionality)

#### Describe the changes you have made in this PR -)
Replaced the invoices lib by bolt11

PS: needs to be tested thoroughly

### CODE REVIEW

**Review by bumi — APPROVED** [review]
utACK

File: src/app/screens/ConfirmPayment/index.tsx [hunk 1]
```diff
@@ -94,8 +93,12 @@ function ConfirmPayment(props: Props) {
           <>
             <div className="mb-8">
               <PaymentSummary
-                amount={invoiceRef.current?.tokens}
-                description={invoiceRef.current?.description}
+                amount={invoiceRef.current?.satoshis}
+                description={
+                  invoiceRef.current?.tags.find(
```
[the remark this answers is not in the document]
Comment (dylancom) [reply src/app/screens/ConfirmPayment/index.tsx:98]: I agree, this feels rather cumbersome. Let's try a PR

[the remark this answers is not in the document]
Comment (dylancom) [reply src/app/screens/ConfirmPayment/index.tsx:98]: I opened a PR: https://github.com/bitcoinjs/bolt11/pull/60

**Comment by dylancom** [comment]
bolt11 decode got updated :)

**Review by escapedcat — APPROVED** [review]
🚀

**Pull request merged**